### PR TITLE
Traitor objective issuers

### PR DESCRIPTION
**Pull request by whateverusername0**
## About the PR

Traitors now know who hired them.

## Why / Balance

Spicing up the briefing text, looks nice imo. Can also be used for RP if anyone cares.

## Technical details

GenerateBriefing now takes objectiveIssuer argument, which if null will be defaulted to "The Syndicate"

## Media

new briefing text
image

if we don't have an employer
image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl: 
- tweak: Traitors now know their employers.

### CODE REVIEW

**Comment by Dutch-VanDerLinde** [comment]
very epic

**Comment by mirrorcult** [comment]
should read more like `You are a traitor sent by (corporation) on behalf of The Syndicate.` imo

**Comment by Brandon-Huu** [comment]
Having traitor in the description is kinda weird.

**Comment by Dutch-VanDerLinde** [comment]
> Having traitor in the description is kinda weird.

well they're traitors are they not?

**Comment by Hrosts** [comment]
> Having traitor in the description is kinda weird.

Doubling on the weirdness thing, though it's not a huge issue.
I think it would be better to separate these into 2 sentences:

> You are a traitor.
CyberSun Industries could use your assistance.

Or something of this sort.

**Comment by Brandon-Huu** [comment]
> > Having traitor in the description is kinda weird.
> 
> well they're traitors are they not?

Traitor would imply that they were working for NT(or whoever) then joined the syndicate.

**Comment by mirrorcult** [comment]
> > > Having traitor in the description is kinda weird.
> > 
> > 
> > well they're traitors are they not?
> 
> Traitor would imply that they were working for NT(or whoever) then joined the syndicate.

the antag is literally called traitor, take it up with whoever added it to ss13 15 years ago or whatever

**Comment by Brandon-Huu** [comment]
> the antag is literally called traitor, take it up with whoever added it to ss13 15 years ago or whatever

Yeah that's its OC name not IC

**Comment by mirrorcult** [comment]
'syndicate agent' as before sounds cleaner than 'traitor' regardless so i guess it doesnt matter

**Comment by Dutch-VanDerLinde** [comment]
> > the antag is literally called traitor, take it up with whoever added it to ss13 15 years ago or whatever
> 
> Yeah that's its OC name not IC

not really they literally are traitors

File: Resources/Locale/en-US/game-ticking/game-presets/preset-traitor.ftl [hunk 1]
```diff
@@ -23,7 +24,7 @@ traitor-death-match-end-round-description-entry = {$originalName}'s PDA, with {$
 
 # TraitorRole
 traitor-role-greeting =
-    You are a syndicate agent.
+    You are a traitor sent from {$corporation}.
```
Comment (mirrorcult) [inline Resources/Locale/en-US/game-ticking/game-presets/preset-traitor.ftl:27]:
```suggestion
    You are an agent sent by {$corporation} on behalf of The Syndicate.
```

**Review by mirrorcult — CHANGES_REQUESTED** [review]

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**